### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by bertysentry** [sentrysoftware/sentry-maven-skin#57]
* Title banner should be in bold, for better readability (if color contrast is not enough)
* BannerLeft and BannerRight images should have a caption
